src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,114 @@
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [title, setTitle] = useState('');
+  const [movie, setMovie] = useState(null);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+
+  }, [title]);
+
+  const getMovie = () => {
+    getData(title)
+      .then((findMovie) => {
+        if (findMovie.Response === 'False') {
+          setError(true);
+          setMovie(null);
+
+          return;
+        }
+
+        const {
+          Title,
+          Plot,
+          Poster,
+          imdbID,
+        } = findMovie;
+
+        setMovie({
+          title: Title,
+          description: Plot,
+          imgUrl: Poster,
+          imdbUrl: `https://www.imdb.com/title/${imdbID}`,
+          imdbId: imdbID,
+        });
+
+        setError(false);
+        setTitle('');
+      });
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={Classnames('input', {
+                'input is-danger': error,
+              })}
+              value={title}
+              onChange={(event) => {
+                setTitle(event.target.value);
+                setError(false);
+              }}
+            />
+          </div>
+          {error && (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
+              className="button is-light"
+              onClick={() => getMovie()}
+            >
+              Find a movie
+            </button>
+          </div>
+
+          <div className="control">
+            <button
+              type="button"
+              className="button is-primary"
+              onClick={() => {
+                addMovie(movie);
+                setMovie(null);
+              }}
+            >
+              Add to the list
+            </button>
+          </div>
         </div>
+      </form>
+
+      <div className="container">
+        <h2 className="title">Preview</h2>
+        {movie && <MovieCard {...movie} />}
```

Comment:
```suggestion
        {movie && (
          <>
            <h2 className="title">Preview</h2>
            <MovieCard {...movie} />}
```
No need to show `preview` label if there's no movie